app/src/server/utils/nodes/startTestJobs.ts

```diff
@@ -113,6 +115,7 @@ export const startTestJobsForEval = async ({
         const secondOutputSourceId = datasetEval.outputSources[j]?.id as string;
         evalsToRun.push({
           nodeEntryId: datasetEvalNodeEntry.nodeEntryId,
+          evaluationModelId: evaluationModelId,
```

Comment:
Nit: this could be replaced with just `evaluationModelId,`

Reply:
Thanks! Will do!